pcweb/pages/index/demos/auth/auth.py

```diff
@@ -1,70 +1,154 @@
-                        weight="bold",
-                        size="3"
-                    ),
+
+auth_code = """class AuthState(rx.State):
+
+    def submit(self, form_data):
+        return rx.toast(form_data)
+
+def auth() -> rx.Component:
+    return rx.card(
+        rx.form(
+            rx.vstack(
+                rx.image(
+                    src="/logo.jpg",
+                    class_name="image",
+                ),
+                rx.heading(
+                    "Create an account",
+                    class_name="heading",
+                ),
+                class_name="vstack",
+            ),
+            rx.vstack(
+                rx.text(
+                    "Email address",
+                    class_name="text",
+                ),
+                rx.input(
+                    placeholder="user@reflex.dev",
+                    name="email",
+                    type="email",
+                    required=True,
+                    class_name="input",
+                ),
+                class_name="vstack",
+            ),
+            rx.vstack(
+                rx.hstack(
                     rx.text(
-                        "Example authentication form.",
-                        size="1"
+                        "Password",
+                        class_name="text",
                     ),
-                    content(),
-                    rx.text(
-                        "Get started with a free account.",
-                        size="1"
+                    rx.link(
+                        "Forgot password?",
+                        href="#",
+                        class_name="link",
                     ),
-                    direction="column",
-                    align_items="center",
-                    width="100%",
-                    spacing="2",
+                    class_name="hstack-password",
                 ),
-                width="20em",
+                rx.input(
+                    placeholder="Enter your password",
+                    name="password",
+                    type="password",
+                    required=True,
+                    class_name="input",
+                ),
+                class_name="vstack",
             ),
-            display="flex",
-            height=demo_height,
-            align_items="center",
-            justify_content="center",
+            rx.button("Sign In", type="submit", class_name="button"),
+            rx.hstack(
+                rx.divider(),
+                rx.text(
+                    "OR CONTINUE WITH",
+                    class_name="text-small",
+                ),
+                rx.divider(),
+                class_name="hstack",
+            ),
+            rx.button(
+                rx.icon(tag="github"),
+                "GitHub",
+                class_name="button-secondary",
+            ),
+            class_name="form",
+            on_submit=AuthState.submit,
         ),
+        class_name="card",
```

Comment:
I think we should add the 
```
app = rx.App()
app.add_page(auth)
```

Only a few more lines but it will give them a fully working app